packages/local_auth/local_auth_platform_interface/lib/types/biometric_type.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Various types of biometric authentication.
+enum BiometricType {
+  /// Face authentication.
+  face,
+
+  /// Fingerprint authentication.
+  fingerprint,
+
+  /// Iris authentication.
+  iris,
+
+  /// Any biometric (e.g. fingerprint, iris, or face) on the device that meets
+  /// or exceeds the requirements for Class 3 (formerly Strong), as defined
+  /// by the Android CDD. Android only.
```

Comment:
We shouldn't make claims about specific platforms in platform interfaces; who knows what future platforms will do. Although it's tricky since these classifications don't seem to be standards-based, just Android.

How about replacing "that meets [...]" with:
> that the platform API considers to be strong. For example, on android this corresponds to Class 3

Then the same for `weak` below.